doc/api-details.md

```diff
@@ -97,14 +102,48 @@ $ curl -X 'POST' \
   -H 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJib2IifQ.ci1smeJeuX779PptTkuaG1SEdkp5M1S1AgYvX8VdB20' \
   -H 'Content-Type: application/json' \
   -d '{
-  "name":"checkout",
-  "revision":{"tree":"mainline",
-  "url":"https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git",
-  "branch":"master",
-  "commit":"2a987e65025e2b79c6d453b78cb5985ac6e5eb26",
-  "describe":"v5.16-rc4-31-g2a987e65025e"}
-}'
-{"_id":"61bda8f2eb1a63d2b7152418","kind":"node","name":"checkout","revision":{"tree":"mainline","url":"https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git","branch":"master","commit":"2a987e65025e2b79c6d453b78cb5985ac6e5eb26","describe":"v5.16-rc4-31-g2a987e65025e"},"parent":null,"status":"pending","result":null, "created":"2022-02-02T11:23:03.157648", "updated":"2022-02-02T11:23:03.157648"}
+    "name": "checkout",
+    "kind": "checkout",
+    "path": ["checkout"],
+    "data": {
+      "kernel_revision": {
+        "tree": "mainline",
+        "url": "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git",
+        "branch": "master",
+        "commit": "2a987e65025e2b79c6d453b78cb5985ac6e5eb26",
+        "describe": "v5.16-rc4-31-g2a987e65025e"
+      }
+    }
+  }' | jq
```

Comment:
I am wondering if we should have one-liner instruction for installing `jq` package as all the sample commands are using it.

Reply:
Yes, I understand that it's optional.
Usually, developers/users follow the instruction guide in the ramp-up phase, and copy-paste commands to try them. In that case, it will generate an error like:
```
Command 'jq' not found, but can be installed with:
...
curl: (23) Failure writing output to destination
```
And then the user will need to check the correct command for installation.
I suggest if we want to keep it in the sample commands, it doesn't hurt to just add a line like `The sample commands below use `jq` package for output formatting and should be installed first to use it.` And we can put a link to its installation guide.

Reply:
@hardboprobot This conversation was still unresolved. Anyway it was just a suggestion :+1: